pegasus/helpers/asset_helpers.rb

```diff
@@ -0,0 +1,26 @@
+require 'sprockets'
+
+# Initializes the asset map from the dashboard assets directory.
+def load_asset_map
+  $asset_map = nil
+  manifest_filename = Sprockets::ManifestUtils.find_directory_manifest(CDO.dashboard_assets_dir)
+  return unless manifest_filename && File.exist?(manifest_filename)
+  json = File.read(manifest_filename)
+  $asset_map = JSON.parse(json)['assets']
+end
+load_asset_map
```

Comment:
You could use a [singleton](http://ruby-doc.org/stdlib-1.9.3/libdoc/singleton/rdoc/Singleton.html) class.

```ruby
class AssetMap
  include Singleton
  
  def initialize
    # load map into member var, @asset_map
  end
  
  # def asset_path, etc...
end
```

Then, 
```ruby
def asset_path(asset)
  AssetMap.instance.asset_path(asset)
end
```

This will lazy-load the first time you call `AssetMap.instance`. If you want to explicitly load on startup, you could simply call `AssetMap.instance` directly.